.semaphore/release/release.yml

```diff
@@ -2,32 +2,44 @@ version: v1.0
 name: Publish official release
 agent:
   machine:
-    type: e1-standard-4
+    type: e1-standard-8
     os_image: ubuntu2004
 
 execution_time_limit:
-  minutes: 600
+  minutes: 800
 
 blocks:
   - name: "Publish official release"
+    dependencies: []
     skip:
       # Only run on branches, not PRs.
       when: "branch !~ '.+'"
     task:
       secrets:
-      - name: quay-robot-calico+semaphoreci
-      - name: docker
-      - name: oss-release-secrets
-      - name: google-service-account-for-gce
-      - name: openstack-signing-publishing
+        - name: quay-robot-calico+semaphoreci
```

Comment:
All of the other pre-existing lists don't use indentation like this - I think we should keep consistent with that.